### PR TITLE
homework1

### PR DESCRIPTION
**Pull request by KirillKlimenok**
Please, Vadim review my code

### CODE REVIEW

File: homework1/task1/task1.cpp [hunk 1]
```diff
@@ -4,5 +4,49 @@ using namespace std;
...
 {
-    
+	int nMutants = 0, dayCounter = 0;
+	short procentInNight = 0;
+	int killInDay = 0;
+	bool isDayOrNight = true;
+
+	cout << "Please enter number of mutants: ";
+	cin >> nMutants;
+	cout << "\nPlease enter procent in night: ";
```
Comment (VadzimAlkhimionak) [inline homework1/task1/task1.cpp]: use `<< endl`

File: homework1/task1/task1.cpp [hunk 2]
```diff
@@ -4,5 +4,49 @@ using namespace std;
...
+	cin >> procentInNight;
+	cout << "\nPlease enter numbers kills in day: ";
```
Comment (VadzimAlkhimionak) [inline homework1/task1/task1.cpp]: use << endl

File: homework1/task1/task1.cpp [hunk 3]
```diff
@@ -4,5 +4,49 @@ using namespace std;
...
+		if ((nMutants * procentInNight / 100) > (killInDay) && (dayCounter == 1)) {
+			cout << "Mutants win";
+			return 0;
+		}
+		else {
+			nMutants += nMutants * procentInNight / 100;
+			isDayOrNight = true;
+			dayCounter++;
+		}
+
```
Comment (VadzimAlkhimionak) [inline homework1/task1/task1.cpp:48]: Remove

**Review by VadzimAlkhimionak — COMMENTED** [review]
Hi, I left some comments. Fix them, please.
Good job! Well done!

File: homework1/task1/task1.cpp (hunk 3 above)
Comment (KirillKlimenok) [reply homework1/task1/task1.cpp:48]: if i'm delete these lines, the program will not work correctly

File: homework1/task1/task1.cpp (hunk 1 above)
Comment (KirillKlimenok) [reply homework1/task1/task1.cpp]: done

File: homework1/task1/task1.cpp (hunk 2 above)
Comment (KirillKlimenok) [reply homework1/task1/task1.cpp]: done

**Pull request merged**